tests/tests/control_diag_debug

```diff
@@ -31,7 +31,6 @@ export FV3_RUN=control_run.IN
 export CCPP_SUITE=FV3_GFS_v16
 export INPUT_NML=control.nml.IN
 
-export FDIAG=0,1,2,3,4,5,6
```

Comment:
This is just clean up since fhmax is only 1 hour?

Reply:
`FDIAG` is no longer used, I think. The control_diag_debug test passes without this setting.